app_flutter/README.md

```diff
@@ -6,4 +6,51 @@
 
 `flutter run -d chrome --web-port=8080`
 
-Must run on port 8080 for Google Sign In to work since that is the only enabled port for localhost.
+Must run on port 8080 for Google Sign In to work since that is the
+only enabled port for localhost.
+
+## Style
+
+### File organization
+
+The `lib` directory is organized in subdirectories as follows:
+
+* `lib/`: main.dart and top-level widgets (e.g. the build dashboard
+  widget). Can import anything.
+
+* `lib/widgets/`: Custom widgets used in this project. The files in
+  this directory should import either the Flutter widgets package or
+  the Flutter material package, and may import any local files except
+  those in the top-level `lib/` directory.
```

Comment:
To clarify, is this saying it can only imports files from `lib/widgets` or is saying do not import files from only the top level of  `lib/`?

Reply:
tried to clarify